.gitignore

```diff
@@ -61,4 +61,7 @@ typings/
 .next
 
 /dist/
+
 yarn.lock
+package.json
```

Comment:
Without this file nothing will work. We need to keep it.

Reply:
I had a feeling that file might be needed :)